src/graphql/resolveConnection.js

```diff
@@ -51,14 +57,33 @@ const resolveConnection = (
       return sql
     })
 
+    // Gets the condition for filtering our result set using a cursor.
+    const getCursorCondition = (cursor, operator) => {
+      const sql = new SQLBuilder()
+
+      const cursorCompareLHS = `(${
+        [orderBy.name, ...primaryKeyNoOrderBy.map(({ name }) => name)]
+          .map(name => `"${name}"`)
+          .join(', ')
+      })`
+
+      const cursorCompareRHS = `(${
+        [null, ...primaryKeyNoOrderBy].map((x, i) => `$${i + 1}`).join(', ')
```

Comment:
I am correct in assuming that the null is a stand in for the orderBy.name? Could it not just be orderBy.name since we only use it to increment the `$1` value.